src/controller.coffee

```diff
@@ -645,6 +657,9 @@ define ['droplet-helper',
   # Simple function for going through a linked-list block
   # and seeing what the innermost child is that we hit.
   Editor::hitTest = (point, block) ->
+    if @readOnly
+      return null
+
```

Comment:
hit testing (in this case) is checking to see if mouse coordinates are considered a "hit" against an "element" or "block" in the UI. By failing all hit tests on droplet's inner elements, it make it impossible to select, drag, set focus, etc.